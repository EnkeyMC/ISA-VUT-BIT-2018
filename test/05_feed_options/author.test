NAME=Show author
ARGS=http://www.mocky.io/v2/5bc8c2f33200008f005a00ee -a
RC=0
CHECK_OUTPUT=1
