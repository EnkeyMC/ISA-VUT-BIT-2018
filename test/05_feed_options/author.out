*** XML.com ***
Processing Inclusions with XSLT
Autor: 

Putting RDF to Work
Autor: Rael Dornfest (mailto:rael@oreilly.com)
